### PR TITLE
Add hostheader

### PR DESCRIPTION
**Pull request by magudb**
This PR is only to ensure that the host header is being set correctly. Sadly it seems that you don't have squash commits enabled on the repos, so i can't make it super pretty.

I have only made changes to:
* httpchecker_test.go
* httpchecker.go

Let me know if you need anything more.

### CODE REVIEW

**Comment by mholt** [comment]
You should be able to just squash commits locally and force-push to your branch... does that not work?

File: cmd/checkup/main.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package main
 
-import "github.com/sourcegraph/checkup/cmd"
+import "github.com/monosolutions/checkup/cmd"
```
Comment (ashfame) [inline cmd/checkup/main.go]: why is this changed?

File: cmd/root.go [hunk 2]
```diff
@@ -7,7 +7,7 @@ import (
 	"log"
 	"os"
 
-	"github.com/sourcegraph/checkup"
+	"github.com/monosolutions/checkup"
```
Comment (ashfame) [inline cmd/root.go]: why is this changed?

Comment (magudb) [reply cmd/root.go]: That is an error, mostly so i can use CheckUp with the changes i needed.

**Comment by magudb** [comment]
creating a new PR
